doc/doxygen.conf.in

```diff
@@ -0,0 +1,2 @@
+EXTRACT_ALL = YES
+EXCLUDE += src
```

Comment:
`EXCLUDE` will still be copied for cross-package docs. I don't think there's any way to trick `lsstDoxygen` about that.

Though looking at the Doxygen documentation, `EXCLUDE_PATTERNS += *astshim/src/*/*.cc` might do the job. As for `afw`, we don't care about stuff getting used if it only matches one package...

Reply:
That sounds perfect. I also agree that in this case we don't really mind if Doxygen scans the source, but this works and seems safe. I'll use the same pattern for afw as well (where we really do want to stop Doxygen from scanning the source, at least some source).